README.md

```diff
@@ -137,6 +142,17 @@ If you'd like to use the old keras version, you can:
 pip install keras==1.2.2 tensorflow
 ```
 
+Finally, to run the most important unit tests, you can use:
+
+```shell
```

Comment:
This is not rendering properly when I click the "view" button above.

Reply:
Fixed